dali/kernels/common/block_setup.h

```diff
@@ -292,15 +317,15 @@ class BlockSetup {
       }
     }
 
-    ret = SetZBlocksPerSample(ret);
+    ret = SetZBlocksPerSample<ndim>(ret);
```

Comment:
Why? This doesn't seem necessary.

Reply:
I think my compiler was complaining, may be due to the intermediate alias.